src/App.css

```diff
@@ -44,6 +44,7 @@ form button:hover {
 .todo-popup-container {
   background-color: rgb(51, 92, 90);
   z-index: 10;
+  position: absolute;
```

Comment:
Note: for creating a popup first decide whether it is going to reside in a container or relative to complete window based on that use the combination of 'relative' and 'absolute' position of container to align the popup.
above will take the window as a relative container for it. @raiab